external-crates/move/crates/move-analyzer/tests/same-pkg-addr-std-dep/Move.toml

```diff
@@ -0,0 +1,15 @@
+# A package with an empty module and the same address as other
```

Comment:
It's a minor thing but with a rather long explanation... Added this package so that we have both the option to depend on this kind of module (with the same address as the module that depends on it) with standard library included and without. Previously (with the standard library omitted from dependencies) we could not open a package both depending on `same-pkg-addr` (no explicit standard library dependency) and  explicitly depending on external crates's standard library in the IDE as it would complain about dependency conflict (implicit vs. explicit). It did not matter for tests (as implicit deps do not kick in in external crates) but was annoying when opening test packages in the IDE